tests/lib/fakestore/store/store.go

```diff
@@ -729,6 +735,14 @@ func (s *Store) snapActionEndpoint(w http.ResponseWriter, req *http.Request) {
 	w.Write(out)
 }
 
+func isAutoRefreshRequest(req *http.Request) bool {
+	refreshReason := req.Header["Snap-Refresh-Reason"]
```

Comment:
Nitpick, can we just use `req.Header.Get("Snap-Refresh-Reason")`?

Reply:
updated